yarn.lock

```diff
@@ -7986,8 +7986,9 @@ eslint-plugin-unicorn@^35.0.0:
     safe-regex "^2.1.1"
     semver "^7.3.5"
 
-"eslint-plugin-va@file:./script/eslint-plugin-va":
-  version "1.0.5"
+"eslint-plugin-va@link:./script/eslint-plugin-va":
+  version "0.0.0"
+  uid ""
```

Comment:
why is this going to version 0.0.0?

Reply:
That happened when I switched the dependency protocol from `file` to `link`.